### PR TITLE
implemented administration page ui

### LINKED ISSUE #47: Administration page UI implementation
### Contact Details

_No response_

### GSSoC-22

yes

### Type of Issue

UI Related

### Description

Administration page UI should be implemented according to the UI. for now use the dummy data.
@chayan-dev will give you the Figma access.

administration_page_ui

### Error Logs

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Vibhuti2002**
Fixed: #47 
implemented administration page ui
Screenshot (285)

### CODE REVIEW

File: app/src/main/res/layout/administration_ui.xml [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+                    android:layout_gravity="center"
+                    android:fontFamily="@font/roboto_bold"
+                    android:gravity="center"
+                    android:text="@string/prof_binay_k_dutta_chariman_west_bengal_pollution_control_board_chairman"
+                    android:textColor="@color/black"
+                    android:textSize="14sp" />
+            </com.google.android.material.card.MaterialCardView>
+
+            <com.google.android.material.card.MaterialCardView
+                android:layout_width="309dp"
```
Comment (logic-cribber) [inline app/src/main/res/layout/administration_ui.xml]: Declare all the dimension in Dimen.xml

File: app/src/main/res/layout/administration_ui.xml [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+                android:id="@+id/txtPageTitle"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="45dp"
+                android:layout_marginTop="47sp"
+                android:layout_marginEnd="131dp"
+                android:fontFamily="@font/roboto_bold"
+                android:text="@string/administration"
+                android:textColor="@color/white"
+                android:textSize="23sp"
```
Comment (logic-cribber) [inline app/src/main/res/layout/administration_ui.xml]: define size in dimen.xml

File: app/src/main/res/layout/administration_ui.xml [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <FrameLayout
+            android:id="@+id/pageTopFl"
+            android:layout_width="match_parent"
+            android:layout_height="181dp"
+            android:alpha="0.7"
+            android:background="@drawable/home_header_college_img">
```
Comment (logic-cribber) [inline app/src/main/res/layout/administration_ui.xml]: Always add svg images in Project

**Review by logic-cribber — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/administration_ui.xml (hunk 2 above)
Comment (HeySitam) [reply app/src/main/res/layout/administration_ui.xml]: yes, bro that's a best practice. Will love, if you work on this issue.

**Pull request merged**